src/olympia/lib/settings_base.py

```diff
@@ -1419,9 +1419,9 @@ def read_only_mode(env):
     env['READ_ONLY'] = True
 
     # Replace the default (master) db with a slave connection.
-    if not env.get('SLAVE_DATABASES'):
+    if not env.get('REPLICA_DATABASES'):
         raise Exception('We need at least one slave database.')
-    slave = env['SLAVE_DATABASES'][0]
+    slave = env['REPLICA_DATABASES'][0]
```

Comment:
note for later: we will have to co-ordinate with ops to update the env variables too.